tests/positive/general/preemption.go

```diff
@@ -75,7 +75,15 @@ func makePreemptTest(components string) types.Test {
 
 	var systemFiles []types.File
 	for _, component := range []string{"b", "u"} {
-		if enabled(component) {
+		if enabled(component) && strings.Compare(component, "b") == 0 {
+			systemFiles = append(systemFiles, types.File{
+				Node: types.Node{
+					Name:      longnames[component] + ".ign",
+					Directory: "base.d",
```

Comment:
It'd be less repetitious to conditionalize the directory rather than the entire append operation.